pyproject.toml

```diff
@@ -58,7 +58,7 @@ warn_unused_configs = true
 
 [tool.ruff]
 # Exclude a variety of commonly ignored directories.
-exclude = ["docs*", "conda*"]
+exclude = ["docs", "conda"]
```

Comment:
I don't think these wildcards are needed and I think there's a small risk of unintended consequences.